DANS-KNAW/easy-manage-deposit
# EASY-1987: easy-manage-deposit geeft ten onrechte aan dat dryad doi's niet geregistreerd zijn

Fixes EASY-1987

#### When applied 
Deposits that 
 - have a DOI not originating from DANS and
 - are ARCHIVED and
 - property `identifier.dans-doi.registered` = no 

don't appear anymore in error reports (dans-doi is not registered, but nevertheless the deposit has a valid DOI)

#### Where should the reviewer @DANS-KNAW/easy start?

#### How should this be manually tested?
- log into `dtap`
- see if in `/var/opt/dans.knaw.nl/tmp/easy-ingest-flow-inbox` or in` /var/opt/dans.knaw.nl/tmp/easy-sword2` you have a deposit with state `ARCHIVED` and a not-DANS DOI.  Otherwise change the property values in one of the deposits.
- run `easy-manage-deposit report error`
- now you should see no deposits with state ARCHIVED in the error report

#### Related pull requests on github
repo                       | PR                | note
-------------------------- | ----------------- | ----
easy-                      | [PR#](PRlink)     | ..
